image_processing/validation.py

```diff
@@ -154,6 +154,8 @@ def check_colour_profiles_match(source_filepath, converted_filepath):
             if source_image.mode != converted_image.mode:
                 if source_image.mode == BITONAL and converted_image.mode == GREYSCALE:
                     logger.info('Converted image is greyscale, not bitonal. This is expected')
+                elif source_image.mode in ['RGBX', 'I;16'] and converted_image.mode in ['RGB', 'RGBA']:
```

Comment:
This exception is only necessary for conversions from TIFF to JPG when creating thumbnails, and we don't usually run this check on the thumbnail, so I'm removing this to keep the validation as stringent as possible